doc/compiler/ProblemDetermination.md

```diff
@@ -133,7 +133,7 @@ contains a potentially faulty optimization. If the program still fails at
 `noOpt`, then the problem is most likely in the code generation phase (versus
 the optimization phase) of the compiler.
```

Comment:
I think it's worth mentioning early on in the document that you can change opt levels by specifying `-Xjit:optlevel=noOpt` command line or setting `TR_Options='optLevel=noOpt'` and then refer to the doc/compiler/CompilerOptions.md file for more details.